### DISCUSSION TITLE
How to set the date/timestamp field to be used for the Calendar Layout

### DISCUSSION THREAD

**Discussion by VaZark** [directus/directus#12321] [Archive]
I have a timestamp field named  "Creation Date" but when I switch to calendar view, and set it as the _Display Template_  the values do not show up in the Calendar. I'm probably missing some steps here. How do I use the Calendar layout effectively?

**Comment by rijkvanzanten** [comment]
Make sure to configure the Start Date Field as well to instruct the calendar what field to use for the dates on the calendar:

![CleanShot 2022-03-23 at 11 02 58@2x](https://user-images.githubusercontent.com/9141017/159730255-6431b7db-8cdf-43d5-a3cb-057daf559bea.png)

**Reply by VaZark** [reply]
It works!! 

Why are they called `start date field` and `end date field`? I thought they were filters over the base date field

**Reply by rijkvanzanten** [reply]
Because an event could have a start and an end date, which allows it to span a longer timeframe (for example meetings might take an hour, or you go on vacation for a week etc). The "Field" that contains the "Start Date" = "Start Date Field"

**Reply by benhaynes** [reply]
Let's get a placeholder in `End Date Field` explaining it's optional... and/or a note explaining that it is used for events that span datetimes.
